biogeochem/FatesAllometryMod.F90

```diff
@@ -1489,7 +1507,111 @@ subroutine dh2blmax_2pwr(d,p1,p2,dbh_maxh,c2b,blmax,dblmaxdd)
     
     return
   end subroutine dh2blmax_2pwr
-  
+
+
+  ! ===========================================================================
+
+
+  subroutine dh2blmax_3pwr(d,h,dhdd,p1,p2,p3,slatop,dbh_maxh,c2b,blmax,dblmaxdd)
+     !--------------------------------------------------------------------------
+     !
+     !     This function calculates the maximum leaf biomass from reference 
+     ! diameter, plant height and top-of-the-canopy (fully sunlit) SLA. This 
+     ! functional form is similar to Lescure et al. (1983) and Longo et al.
+     ! (2020), except that it uses SLA as an additional scaler for the 
+     ! allometric equation that can have a different exponent from 
+     ! (DBH^2 * Height).
+     !
+     ! -----------------
+     !  References
+     ! -----------------
+     ! Lescure JP, Puig H, Riera B, Leclerc D, Beekman A , Beneteau A. 1983.
+     !    La phytomasse epigee d'une foret dense en Guiane Francaise
+     !    Acta Oecol.-Oec. Gen. 4: 237-251.
+     !    URL http://www.documentation.ird.fr/hor/fdi:010005089
+     !
+     ! Longo M, Saatchi SS, Keller M, Bowman KW, Ferraz A, Moorcroft PR,
+     !    Morton D, Bonal D, Brando P, Burban B et al. 2020. Impacts of
+     !    degradation on water, energy, and carbon cycling of the Amazon 
+     !    tropical forests. J. Geophys. Res.-Biogeosci. 125:
+     !    e2020JG005677. doi:10.1029/2020JG005677.
+     !
```

Comment:
The new leaf biomass allometry looks great to me. One minor suggestion is to put suggested parameter values. It looks like the default d2bl1, d2bl2 and d2bl3 will give very high biomass with this equation so some suggested values for users wishing to use this option could be helpful.